src/core/libmaven/Compound.h

```diff
@@ -58,6 +58,10 @@ class Compound{
         string pubchem_id;      /**@param  -  pubchem_id -    PubChem id*/
         string hmdb_id;         /**@param  -  hmdb_id-    Human Metabolome Database id */
         string alias;       /**@param   -  alias name of compound   */
+
+        // TODO: from MAVEN (upstream), find out what this is
+        string smileString;
```

Comment:
smiles is a text notation to store a metabolite's structure. This information is probably present in a .MSP file because it has a lot of compound specific information. Just fyi